### DISCUSSION TITLE
New langfuse integration with Pythonic LLM library Mirascope

### DISCUSSION THREAD

**Discussion by clemra** [langfuse/langfuse#2080] [Announcements]
Hi everyone, 

Happy to announce our latest integration with the new and very Python-native project [Mirascope](https://github.com/Mirascope/mirascope). 

The integration works elegantly with the 'with_langfuse' decorator and logs rich traces back to Langfuse as you write the Python code you're already used to. 

Head over to our [changelog](https://langfuse.com/changelog/2024-05-16-mirascope-integration) or go and see a neat explanatory video on the [Mirascope blog](https://www.mirascope.io/post/langfuse-integration). 

Happy building!

**Comment by willbakst** [comment]
If you have any questions about [Mirascope](https://github.com/Mirascope/mirascope) in particular, feel free to ask me!

Best way to reach me would be to DM me in our [Slack Community](https://docs.mirascope.io/latest/HELP/#slack) or post an Issue on our GitHub :)
